packages/errors/src/codes.ts

```diff
@@ -13,6 +13,8 @@ export const SOLANA_ERROR__INVALID_KEYPAIR_BYTES = 4 as const;
 export const SOLANA_ERROR__BLOCK_HEIGHT_EXCEEDED = 5 as const;
 export const SOLANA_ERROR__NONCE_INVALID = 6 as const;
 export const SOLANA_ERROR__NONCE_ACCOUNT_NOT_FOUND = 7 as const;
+export const SOLANA_ERROR__EXPECTED_INSTRUCTION_TO_HAVE_ACCOUNTS = 70 as const;
+export const SOLANA_ERROR__EXPECTED_INSTRUCTION_TO_HAVE_DATA = 71 as const;
```

Comment:
Trying to avoid merge conflict wars are we? 😄

Reply:
I saw your feedback PR had errors up to 51, and then Steve's latest jumped to 99901, so I figured I'd go somewhere in the middle 😄

Reply:
I've just resigned myself to rebasing on top of everyone else's numbers. I don't mind.